.travis.yml

```diff
@@ -13,13 +13,13 @@ matrix:
       env: INTERP=py38
     - python: "3.9"
       env: INTERP=py39
-    - python: "pypy3"
+    - python: "pypy3.6-7.3.1"
       env: INTERP=pypy3
 before_script:
   # Disable IPv6. Ref travis-ci/travis-ci#8711
   - echo 0 | sudo tee /proc/sys/net/ipv6/conf/all/disable_ipv6
 script:
-  - tox -e $INTERP-nocov,$INTERP-cov,qa,docs
+  - tox -e qa && tox -e docs && tox -e $INTERP-cov
```

Comment:
Shortens time taken to run on Travis CI, so we can be frugal with however much "free minutes" we have left. I'm setting up GitHub Actions in the meanwhilst; see 3b10324